lib/plugins/aws/info/display.js

```diff
@@ -68,11 +68,15 @@ module.exports = {
 
     message += functionsMessage;
 
-    // when verbose info is requested, add the stack outputs to the output
-    if (this.options.verbose) {
+    const hasCustomOutput = _.some(this.gatheredData.outputs, output => output.CustomOutput);
+
+    // when verbose info is requested, display all the stack outputs
+    if (hasCustomOutput || this.options.verbose) {
```

Comment:
Do we need the `verbose` check here? Seems like they'll be displayed anyway when there are some outputs.